app/assets/javascripts/application.js

```diff
@@ -1,3 +1,16 @@
+// DO NOT REQUIRE jQuery or jQuery-ujs in this file!
```

Comment:
We're using webpack for jQuery and jQuery-ujs. See the `client/package.json` file.